test/integration/integration.h

```diff
@@ -207,11 +207,12 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
   void createGeneratedApiTestServer(const std::string& bootstrap_path,
                                     const std::vector<std::string>& port_names,
                                     bool allow_unknown_static_fields,
-                                    bool reject_unknown_dynamic_fields, bool allow_lds_rejection);
+                                    bool reject_unknown_dynamic_fields,
+                                    bool ignore_unknown_dynamic_fields, bool allow_lds_rejection);
```

Comment:
We now have 3 of these unknown field setting that are passed around, time for a `struct` wrapper?

Reply:
Done, although I am less than perfectly confident the scoping is ideal.